<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adresse</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_street">
       <property name="text">
        <string>&amp;Stra√üe</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_street</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_street"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_house_number">
       <property name="text">
        <string>&amp;Hausnummer</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_house_number</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_house_number"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_city">
       <property name="text">
        <string>&amp;Stadt</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_city</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_city"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_state">
       <property name="text">
        <string>&amp;Bundesland</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_state</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_state"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_zipcode">
       <property name="text">
        <string>&amp;Postleitzahl</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_zipcode</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_zipcode"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_country">
       <property name="text">
        <string>&amp;Land</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_country</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_country"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_uid">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>5</pointsize>
      </font>
     </property>
     <property name="text">
      <string>uid: -</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
